packages/common/src/intl/locales/en/distribution.json

```diff
@@ -46,6 +46,7 @@
   "label.units-in-pack-size-of_one": "unit in pack size of",
   "label.units-in-pack-size-of_other": "units in pack size of",
   "messages.cant-delete-invoices": "Can only delete invoices with a status of 'New' or 'Allocated'",
+  "messages.no-stock-available": "There is no stock available",
```

Comment:
A bit of a placeholder. This is showing when the status is allocated+ and there is no stock.

It was showing no rows and the auto-allocate input etc

![image](https://user-images.githubusercontent.com/35858975/157192827-8bcd1477-81e7-49d0-bba2-f11132f297fc.png)

Maybe better would be to actually offer a solution.. maybe an an 'order more' button or even a list of links

- Order more
- Create inventory adjustment

Or maybe even an input to easily create an invad through this modal

Reply:
Good idea - have added as a possible requirement for v1+